### ISSUE TITLE
GPL kills

### ISSUE THREAD

**Issue by howardroark2018** [Consunet/Apps#40]
This is an example of a great project that got ruined by the Shitty GPL license! I wished it was MIT or Apache 2.0 or even BSD! 
:( sad!

**Comment by kubakk** [comment]
Could you provide some insight into how the GPL prevents you from contributing to or using this project?

**Comment by sheafferusa** [comment]
Hello boss,

Given below are a few points which support my opinions:

Top 3 Reasons Why An MIT license is Much Better Than a GNU/GPL license:

A GNU/GPL license for a programme or software that runs at the client side, forces the commercial developers to open source every bit of their own source code of the entire software even when what they used under GPL was only 1% of the total code base of the resulting software. This specifically discourages the commercial software dev companies to use GPL at client side. This problem is solved completely in MIT or BSD or even Apache 2.0.

GNU/GPL licenses have been completely banned by both Apple and Microsoft as regards their mobile platforms. Apple has strictly stated in their terms & conditions licensing agreement that, a GPL license (Including all other copyleft licenses ) is in-compatible with iOS’s terms & conditions (According to DRM or Digital Rights Management terms), and thus this has resulted in the rejection of many world class famous softwares/apps, from Apple’s App Store; two of them being —VLC Media Player, —Mozilla Firefox. The same is true with the latest version of Windows 8.1 & above (For the windows App Store) In contrast to GPL, the other licenses like the MIT license, are perfectly compatible with both of the platforms, i.e.; Apple’s iOS as well as Microsoft’s Windows 8.1 Mobile platform (I explicitly mean their respective App Stores.) This problem hampers the growth of the aoftwares under these licenses and hundreds of millions of users have no choice but to plainly ignore such softwares. 

MIT License grants the users as well as developers, the maximum rights and freedom. These licenses are also approved by the various licensing regulating bodies viz.

• FSF approved (Free Software Foundation)

• OSI approved (Open Source License)

• DFSG compatible ( The Debian Free Software Guidelines)

• An MIT license is even 'GPL Compatible'.

Some further references:

http://lifehacker.com/5728521/apple-pulls-vlc-from-the-ios-app-store

http://en.wikipedia.org/wiki/License_compatibility

http://www.tuaw.com/2011/01/09/the-gpl-the-app-store-and-you/

http://en.wikipedia.org/wiki/Open-source_license#OSI_approved_licenses

That's it! I hope now, I sound much more convincing to you. Thanks a lot for taking out your valuable time to go through this lengthy reply of mine. (I almost did my best to get my points across to you in the shortest way I possibly could.)

-Anime.sh

**Comment by kubakk** [comment]
My interpretation of your argument is that GPL restricts commercial exploitation. I'd be happy to license our software on commercial terms when someone or a company requests this. I don't think the open source community benefits from commercial exploitation of code without profit sharing with the original authors. The GPL allows everyone to see what the product is, use it in other free projects and requires the third party developers to negotiate a commercial license if the code is used in a for-profit product that isn't suitable for GPL.

In short, code can be managed under multiple licenses. We think the GPL gives the best mix of openness and protection when making it available here. That doesn't mean that you can't have this code licensed in a different way for your specific purpose.
